src/robot/running/context.py

```diff
@@ -22,6 +23,7 @@ class ExecutionContexts:
 
     def __init__(self):
         self._contexts = []
+        self.event_loop_ref = None
```

Comment:
I don't think this is needed. It ought to be enough if actual context objects have the reference.